### PR TITLE
Config globbing

### PR DESCRIPTION
**Pull request by tokee**
Uses the newly implemented globbing capabilities for YAML loading.

Also adds support for resolving globs relative to the class path as that is a common case when resolving configurations.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 * Added `YAML.parse` methods that takes a Path or File as input, rather than a classpath Name or InputStream. Just for more flexibility in how to read your YAML file.
+* Added globbing (rsolving of files with wildcards) to the Resolver
```
Comment (sbktc) [inline CHANGELOG.md:10]: Typo: "rsolving" shouldn't it be "resolving"?

File: src/main/java/dk/kb/util/yaml/YAML.java [hunk 2]
```diff
@@ -672,16 +675,23 @@ public static YAML parse(File yamlFile) throws IOException {
      * Note: This method merges the YAML configs as-is: Any key-collisions are handled implicitly by keeping the latest
      * key-value pair in the stated configurations.
      *
-     * @param configNames the names of the configuration files.
+     * Note 2: The resolver supports globbing so {@code /home/someone/myapp-conf/*.yaml} expands to all YAML-files
+     * in the {@code myapp} folder. When globbing is used, the matching files are returned in alphanumerical order.
```
Comment (sbktc) [inline src/main/java/dk/kb/util/yaml/YAML.java:679]: myapp-conf folder

File: src/main/java/dk/kb/util/yaml/YAML.java [hunk 3]
```diff
@@ -672,16 +675,23 @@ public static YAML parse(File yamlFile) throws IOException {
...
         List<InputStream> configs = null;
         try {
-            configs = Arrays.stream(configNames).map(Resolver::resolveStream).collect(Collectors.toList());
+            configs = Arrays.stream(configNames).
+                    map(Resolver::resolveGlob).flatMap(Collection::stream).
+                    map(YAML::openStream).
+                    collect(Collectors.toList());
             InputStream yamlStream = null;
             for (InputStream config : configs) {
+                System.out.println("Streamo");
```
Comment (sbktc) [inline src/main/java/dk/kb/util/yaml/YAML.java:694]: Debugging leftover?

File: src/main/java/dk/kb/util/yaml/YAML.java [hunk 4]
```diff
@@ -694,11 +704,23 @@ public static YAML resolveMultiConfig(String... configNames) throws IOException
             }
         }
     }
+    private static InputStream openStream(Path path) {
+        try {
+            System.out.println("Opening " + path);
```
Comment (sbktc) [inline src/main/java/dk/kb/util/yaml/YAML.java:709]: Replace with log.debug perhaps?

**Review by sbktc — CHANGES_REQUESTED** [review]

File: src/main/java/dk/kb/util/yaml/YAML.java (hunk 4 above)
Comment (tokee) [reply src/main/java/dk/kb/util/yaml/YAML.java]: Nah, it already log.debugs the list of resolved paths. It was just a leftover from locating a problem.

**Pull request merged**